### ISSUE TITLE
Improve accessibility in Skills

### ISSUE THREAD

**Issue by schiltpuzzle** [puzzle/skills#1201]
**Description of feature or bug**
When you go to Skills and press `F12` and then go the the accessibility tab and select `all issues` under the drop-down `check for issues` you will find lots errors. Most of them are very easy fixes (Missing text labels, contrast, ...). Sometimes you have to check if the error has to be fixed. We don't want to achieve the accessibility standard `AAA`. Skills currently doesn't use any of the `html Semantic tags`. These Tags are very useful for accessibility.

Maybe Install the Browser Plugin `WAVE Web Accessibility Evaluation Tools` and run it on Skills. This Plugin shows Detailed Accessibility Errors. 

**TODO**
- [ ] Find the accessibility errors
- [ ] Check if they make sense
- [ ] Add `html Semantic tags`
- [ ] Create follow up  tickets if the fix exceeds the scope.
